src/client.ts

```diff
@@ -184,7 +184,7 @@ export class Turbopuffer {
   region: string | null;
   defaultNamespace: string | null;
 
-  baseURL: string;
```

Comment:
Same as Python: allowing changing the `base_url` after client construction seems not very important and messes with the region substitution, so I just removed it.